src/admin/includes/formDefinitions/form_buildings.php

```diff
@@ -5,6 +5,12 @@
     'table' => "building"
 ));
 
+if(!is_empty($_POST) || session::has('POST')) {
+    $processor = formBuilder::createProcessor();
```

Comment:
You shouldn't have to include these on each form. It is redundant code that can be consolidated.